src/client/src/components/App.js

```diff
@@ -125,7 +125,7 @@ class App extends Component {
                     />
                     <Route
                       exact
-                      path="/participants/date/details"
+                      path="/participant/:id/date/details/:id"
```

Comment:
Here again, I think you'll need to change one of the `:id`'s into something else in order to have access to them in the component.

Reply:
I will solve this wrong